### PR TITLE
Adding PySD

### PR DESCRIPTION
**Pull request by enekomartinmartinez**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by enekomartinmartinez** [comment]
@conda-forge/help-python hi,
I am having some conflicts when building the packages. The package is updated to PyPI and working fine, I created the recipe using grayskull. However, I don't know with the building fail, as all the dependencies are already satisfied in conda-forge.

I also wanted to ask, as far as the package is in PyPI it will automatically update as soon as a new release is uploaded to PyPI? Or do I need to PR new versions or changes in the requirements to the recipe?

**Comment by enekomartinmartinez** [comment]
@conda-forge-admin, please ping team

**Comment by JamesPHoughton** [comment]
looks good to me. =) Thanks!

File: recipes/pysd/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    - lxml
+    - numpy
+    - openpyxl
+    - pandas
+    - parsimonious
+    - pydoe
+    - regex
+    - xarray
+    - xlrd
+    - xlwt ==1.2.0
```
Comment (dopplershift) [inline recipes/pysd/meta.yaml:36]: I think this is a typo that is causing the builds to fail due to unsatisfiable dependencies:
```suggestion
    - xlrd ==1.2.0
    - xlwt
```

File: recipes/pysd/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pysd-{{ version }}.tar.gz
+  sha256: d6c759a8a64ff3a7df939c9e4aaba57cf6bbffbdf46f1bc1dc14b3c3c59b2860
+
+build:
+  number: 0
+  skip: true  # [py<37]
```
Comment (dopplershift) [inline recipes/pysd/meta.yaml:15]: This looks like a pure Python package that can be noarch: python:
```suggestion
  noarch: python
```

File: recipes/pysd/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - python
```
Comment (dopplershift) [inline recipes/pysd/meta.yaml:20]: If noarch python, then this should be:
```suggestion
    - python >=3.7
```

File: recipes/pysd/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+    - pip
+  run:
+    - python
```
Comment (dopplershift) [inline recipes/pysd/meta.yaml:23]: As above:
```suggestion
    - python >=3.7
```

**Review by dopplershift — CHANGES_REQUESTED** [review]

File: recipes/pysd/meta.yaml (hunk 1 above)
Comment (enekomartinmartinez) [reply recipes/pysd/meta.yaml:36]: Thank you, seems that I made this error when I move the dependencies to `requirements.txt` :S
As with the last version of PySD we remove some requirements, I will update the package on PyPI and then modify also the yaml file :)

**Comment by enekomartinmartinez** [comment]
requirements.txt wasn't added to the source distribution. I included it in PySD's MANIFEST.in to have included it in the next release.

**Review by dopplershift — APPROVED** [review]

**Comment by dopplershift** [comment]
@enekomartinmartinez Next release you should probably update `MANIFEST.in` to include `LICENSE` in the source distribution, then you could remove it from the recipe.

**Pull request merged**